nda_caselist_filter.py

```diff
@@ -0,0 +1,321 @@
+#!/usr/bin/env python
+"""
+Filters ndar_subject01 based on the following selection criteria:
+
+NDA Release 3:
+1. The participant has to be either CHR or HC
+2. If we upload only included subjects, subjects have to fulfill inclusion criteria
+3. Additionally, the inclusion form has no QC issues
+4. If the participant is a CHR,
+    the participant has to fulfill the criteria for the SIPS and/or CAARMS group.
+5. If the participant is a HC,
+    the participant does not fulfill criteria ofany of the SIPS and CAARMS group.
+6. Additionally, the screening PSYCHS forms have no QC issues
+
+5 and 6 can be simplified to:
+    Inconsistecy between the SIPS/CAARMS group and the participant's cohort
+"""
+import json
+import logging
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Tuple
+
+import pandas as pd
+
+logger = logging.getLogger(__name__)
+logargs = {
+    "level": logging.DEBUG,
+    # "format": "%(asctime)s - %(process)d - %(name)s - %(levelname)s - %(message)s",
+    "format": "%(asctime)s - %(process)d - %(name)s - %(filename)s:%(lineno)d - %(levelname)s - "
+    "%(message)s",
```

Comment:
Wasn't the commented line a better (concise) log?

Reply:
The added `%(filename)s:%(lineno)d` information was helpful during debugging. I can switch over to the more concise logs!